modules/environment/aws/code-services/main.tf

```diff
@@ -202,3 +202,47 @@ resource "aws_sqs_queue_policy" "code_services_queue_policy" {
 }
 POLICY
 }
+
+resource "aws_iam_role" "sqs_role" {
+  count  = var.enable_aws_code_services ? 1 : 0
+  name = "${var.cluster}_sqs_role"
+
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "sqs:*",
+                "sts:*"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        },
+    ]
+}
+EOF
+
+  permissions_boundary = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/${aws_iam_policy.sqs_role_boundary.name}"
```

Comment:
this shouldn't be a new policy..just use the developer policy

Reply:
I didn't know there was a developer policy, just added it